MapboxNavigation/MapboxSpeechController.swift

```diff
@@ -98,7 +108,10 @@ public class MapboxVoiceController: RouteVoiceController {
         super.speak(instruction)
     }
     
-    func fetch(instruction: SpokenInstruction) {
+    /**
+     Fetches and plays an instruction.
+     */
+    @objc open func fetch(instruction: SpokenInstruction) {
```

Comment:
Set the argument’s label to `_ instruction`, so that the call site reads more naturally as “fetch modified instruction” instead of “fetch instruction modified instruction”.